src/test/java/seedu/condonery/logic/commands/EditPropertyCommandTest.java

```diff
@@ -141,20 +142,23 @@ public void execute_invalidPersonIndexFilteredList_failure() {
         // ensures that outOfBoundIndex is still in bounds of address book list
         assertTrue(outOfBoundIndex.getZeroBased() < model.getPropertyDirectory().getPropertyList().size());
 
-        EditCommand editCommand = new EditCommand(outOfBoundIndex,
+        EditPropertyCommand editCommand = new EditPropertyCommand(outOfBoundIndex,
             new EditPropertyDescriptorBuilder().withName(CommandTestUtil.VALID_NAME_BOB).build());
 
         CommandTestUtil.assertCommandFailure(editCommand, model, Messages.MESSAGE_INVALID_PROPERTY_DISPLAYED_INDEX);
     }
 
     @Test
     public void equals() {
-        final EditCommand standardCommand = new EditCommand(INDEX_FIRST_PROPERTY, CommandTestUtil.DESC_AMY);
+        final EditPropertyCommand standardCommand =
+                new EditPropertyCommand(INDEX_FIRST_PROPERTY, CommandTestUtil.DESC_AMY);
 
         // same values -> returns true
         EditPropertyDescriptor copyDescriptor = new EditPropertyDescriptor(CommandTestUtil.DESC_AMY);
-        EditCommand commandWithSameValues = new EditCommand(INDEX_FIRST_PROPERTY, copyDescriptor);
-        assertTrue(standardCommand.equals(commandWithSameValues));
+        EditPropertyCommand commandWithSameValues =
+                new EditPropertyCommand(INDEX_FIRST_PROPERTY, copyDescriptor);
+        assertTrue(standardCommand.getEditPropertyDescriptor()
+                .equals(commandWithSameValues.getEditPropertyDescriptor()));
```

Comment:
any reason why this testcase was changed? instead of just calling ::equals() on the EditPropertyCommands